### DISCUSSION TITLE
Using `zenity --progress` with nu

### DISCUSSION THREAD

**Discussion by N4tus** [nushell/nushell#10167] [Q&A]
`zenity progress` displays a progress indicator that is advanced by printing the new progress to its stdin. I can imperatively use `print` to print the progress and `sleep` to simulate work, but if i pipe a command to `zenity --progress` then zenity will only start once the command before finishes, and the would need to be run in parallel.  
Is it possible to run to commands side by side wither their stdout/stdin connected?

**Comment by amtoine** [comment]
> the would need to be run in parallel. Is it possible to run to commands side by side wither their stdout/stdin connected?
>
unfortunately, we do not have background / parallel task control for now :cry: 
### :point_right: see https://github.com/nushell/nushell/issues/247

**Comment by orivej** [comment]
Currently this works as expected: `1..100 | each {|i| sleep 100ms; $i } | to text | zenity --progress`
